src/server/pkg/hashtree/db.go

```diff
@@ -0,0 +1,1703 @@
+package hashtree
```

Comment:
So we're still using the bolt hashtree in a few places I saw in the code. Would it be hard to remove those? Would be nice to remove this code, but that might be too hard  right now.

Reply:
That would be way to much right now. The input repos depend on that and have random read/write semantics.